process/smartContract/scQueryService_test.go

```diff
@@ -150,6 +153,75 @@ func TestExecuteQuery_EmptyFunctionShouldErr(t *testing.T) {
 	assert.Equal(t, process.ErrEmptyFunctionName, err)
 }
 
+func TestExecuteQuery_ShouldPerformActionsInRegardsToAllowanceChannel(t *testing.T) {
+	t.Parallel()
+
+	chanAllowedQueries := make(chan struct{}, 1)
```

Comment:
In `nodeRunner.go`, the new channel is an unbuffered one. Should the test also create an unbuffered channel, to be more accurate?

Reply:
right. after some refactoring, I've forgot to update this test. done